web3deployment/retrieveArtifacts.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const util = require('util');
+const readdir = util.promisify(fs.readdir);
+const path = require("path");
+const artifactsPath = path.join(__dirname, "../artifacts/");
+const buidlerConfigSol5 = path.join(__dirname, "../buidlerConfigSol5.js");
+const buidlerConfigSol4 = path.join(__dirname, "../buidlerConfigSol4.js");
+const execSync = require('child_process').execSync;
+
+module.exports.retrieveArtifacts = main;
+async function main(skipCompilation) {
+  if (!skipCompilation) {
```

Comment:
where is this defined?

Reply:
For now, no, but if you've already done the compilation before, can pass in a `true` value. Just a nice to have